### PR TITLE
added image & input component

### PR DESCRIPTION
**Pull request by irajbhinde**
Hey! Please review my code & let me know if any changes are to be made

Added Components -

**Image** 

1. Responsive Image
2. Round Images

**Input** 

1. Input Textbox
2. Input Textbox with Validation
3. Input Passwords
4. Input Type Radio

**GIFY -**
image_and_input

### CODE REVIEW

File: Components/Input/input.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+          Input textbox can be used to take information from user such as Email
+          Id, phone number, address & other descriptions
+        </div>
+        <pre>
+                <code class="plaintext">
+                    &lt;div class="input-container"&gt;
+                    &lt;p class="input-title">Username*&lt;/p&gt;
+                    &lt;input class="input-field" type="text"&gt;
+                    &lt;/div&gt;
+                </code>
```
Comment (darshandamre) [inline Components/Input/input.html:74]: one tip here, if you want to add indentation to a line you can add `&emsp;` at the start of that line

**Review by darshandamre — COMMENTED** [review]
i think rest of the things are fine

File: Components/Input/input.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.input-title.error {
```
Comment (ivinayakg) [inline Components/Input/input.css:1]: instead of this try adding a class like .input-title--error

File: Components/image/image.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
+
+.responsive{
+  max-width: 100%;
+  height: auto;
```
Comment (ivinayakg) [inline Components/image/image.css:4]: instead of writing auto try using object-fit : contain / cover, will be better for responsivness

**Review by ivinayakg — APPROVED** [review]
code looks code go ahead and merge it, you can think about the suggestions in the future

**Pull request merged**